client/src/components/messages/Conversations.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import SendMessage from "./SendMessage";
+
+const Conversations = ({ message }) => {
+  return (
+    <div className="overflow-x-hidden w-full relative">
+      <div className="bg-slate-50  flex flex-col px-4 pt-20 overflow-x-hidden h-chatScreenHeight">
+        <span className="text-center text-xs my-2">January 15 3:00pm</span>
+        <div className="">
+          <div className="hidden">
+            <img src="" alt="" />
+          </div>
+          <div className="flex inline">
```

Comment:
I think we could delete inline, flex and inline will compete for the same property. There's a different class called inline-flex in Tailwind? I think it looks good just with flex though

Reply:
Yeah, the flex is "Display: flex" then inline is to make the element and inline element but I guess we can achieve same thing with inline-flex

Reply:
Inline-flex is a specific property. If we have 'flex' and 'inline' separately one overrides and cancels the other as they are different values.

Reply:
We can just remove the inline and leave the flex